views/aboutStyles.css

```diff
@@ -1,31 +1,28 @@
-/*------------------------------ header */
-.header-subcontent p{
+/* ------------------------------ header */
+.header-subcontent p {
```

Comment:
- [ ] Please The pages should look almost identical to the original design, outside the content differences, the title color doesn't match the one in the design.
![image](https://user-images.githubusercontent.com/74744787/226133894-262b7b54-4ace-4e7d-b240-e2330b02c6ad.png)
The design
![image](https://user-images.githubusercontent.com/74744787/226133911-883b819a-7a3c-4f3e-80ca-3cae346dedec.png)